utils/chunk-tests.js

```diff
@@ -16,17 +16,17 @@ const runnersMap = new Map([
   ['test-dev', { min: 1, max: 5, runners: ['ubuntu-latest', 'macos-latest'] }],
 ]);
 
-const optionsOverrides = {
-  examples: { min: 1, max: 1 },
+const packageOptionsOverrides = {
+  // 'some-package': { min: 1, max: 1 },
```

Comment:
I left this in for easy use later. In production code, I wouldn't leave this, but for a testing script, I think it's fine.

I could remove it entirely, though.